app/assets/javascripts/components/standings.es6.jsx

```diff
@@ -1,4 +1,27 @@
 class Standings extends React.Component {
+    componentWillMount () {
+        this.setState({week: 'all'})
```

Comment:
I can explain this in person, but basically React components have a concept of `state`, and if that state is ever modified the component will rerender. The render logic will check the week and sort/populate the user record components appropriately - so all we're really doing is setting state to `all` as a default, and letting users change that state via a dropdown